website/docs/best-practices/how-we-style/1-how-we-style-our-dbt-models.md

```diff
@@ -8,6 +8,11 @@ id: 1-how-we-style-our-dbt-models
 - 👥 Models should be pluralized, for example, `customers`, `orders`, `products`.
 - 🔑 Each model should have a primary key.
 - 🔑 The primary key of a model should be named `<object>_id`, for example, `account_id`. This makes it easier to know what `id` is being referenced in downstream joined models.
+- Use underscores for naming dbt models; avoid dots.
+  - ✅  `models_without_dots`
+  - ❌ `models.with.dots`
+  - Use underscores because they’re more compatible and clear across platforms. Dots can cause
+ dbt Cloud parser, quoting, and case sensitivity issues, especially in systems like Snowflake. Databases may also misinterpret dots as schema or database name separators. [dbt uses dots](https://discourse.getdbt.com/t/why-cant-model-names-contain-dots/422) in its `unique_id` generation to differentiate resource types, package names, and resource names, which is another reason to avoid dots in model name. Read the [discussion](https://github.com/dbt-labs/dbt-core/issues/3246) for more details.
```

Comment:
```suggestion
  - Most data platforms use dots to separate `database.schema.object`, so using underscores instead of dots reduces your need for [quoting](/reference/resource-properties/quoting) as well as the risk of issues in certain parts of dbt Cloud. For more background, refer to [this GitHub issue](https://github.com/dbt-labs/dbt-core/issues/3246).
```

Reply:
thanks for tidying this up! i think it might help to start off with the guidance of 'use underscores for model names..' and then continue with the why. 

i folded both in, what do you think about this:

`- Use underscores for model names, not dots. Most data platforms use dots to separate `database.schema.object`, so using underscores instead of dots reduces your need for [quoting](/reference/resource-properties/quoting) as well as the risk of issues in certain parts of dbt Cloud. For more background, refer to [this GitHub issue](https://github.com/dbt-labs/dbt-core/issues/3246).`

Reply:
You already have 
>- Use underscores for naming dbt models; avoid dots.
>    - ✅  `models_without_dots`
>    - ❌ `models.with.dots`

So I didn't think it needed repeating a third time in the context section

Reply:
Oh i see, i saw it was red so I thought it was removed! Got it

Reply:
![image](https://github.com/dbt-labs/docs.getdbt.com/assets/7335046/c21cd381-6472-449e-9dba-ca242033f60b)
For me the only red section is the context section, which is red because it was accidentally spilling onto two lines and I pulled it back to one